README.md

```diff
@@ -34,6 +34,10 @@ To solve the preceding issues, this tool works as follows.
 
 * This tool does not guarantee the atomicity of deletion. If you access the rows that are being deleted, you will get the inconsistent view of the database.
 * This tool does not delete rows which were inserted while the tool was running.
+* This tool does not support truncating tables that use foreign key constraints in some scenarios:
+  * If there is a circular dependency among the tables, truncation will be failed.
+  * If `--tables` is used for a referenced table with ON DELETE CASCADE, the table will be truncated with the referencing tables.
```

Comment:
Sorry for different feedback, but `referenced table with ON DELETE CASCADE` would be somewhat confusing.

Let's say, we have two tables, `Customers` and `ShoppingCarts` (borrowed from [doc](https://cloud.google.com/spanner/docs/foreign-keys/overview#how-to-define-foreign-key-action)), and `ShoppingCarts` table is referencing `Customers` table with `ON DELETE CASCADE`.

In this example, `ShoppingCarts` is the referencing table and `Customers` is the referenced table.

If user specifies `--tables=Customers` (referenced table), then `ShoppingCarts` (referencing table) is also truncated as a effect of `ON DELETE CASCADE`.

So maybe the following might be clearer? `If --tables is used for a table that is referenced by other tables with ON DELETE CASCADE, such tables will also be truncated.`

Reply:
> If --tables is used for a table that is referenced by other tables with ON DELETE CASCADE, such tables will also be truncated.

This sounds more clear! Thank you.

> referenced table with ON DELETE CASCADE would be somewhat confusing.

If I understand correctly, your point is that the foreign key constraint is placed on the referencing table, not on the table being referenced, right?

Reply:
> If I understand correctly, your point is that the foreign key constraint is placed on the referencing table, not on the referenced table, right?

Yes, that's right.